roles/backup-server/tasks/hostkeys.yml

```diff
@@ -0,0 +1,14 @@
+---
+- block:
+  - name: Collect client hostkey
+    command: "{{ 'ssh-keyscan -t rsa ' + item }}"
+    register: clienthostkey
+    changed_when: False
+
+  - name: Add client hostkey to backup server
+    become: true
+    become_user: "{{ backup_user }}"
```

Comment:
Typically I like to see these task controls come after the module and args, but that's a nit-pick.